src/main/java/org/folio/innreach/external/client/feign/InnReachContributionClient.java

```diff
@@ -16,10 +16,11 @@
 import org.springframework.web.bind.annotation.RequestHeader;
 
 import org.folio.innreach.dto.BibInfo;
+import org.folio.innreach.external.client.feign.config.InnReachFeignClientConfig;
 import org.folio.innreach.external.dto.BibItemsInfo;
 import org.folio.innreach.external.dto.InnReachResponse;
 
-@FeignClient(value = "innReachContribution")
+@FeignClient(value = "innReachContribution", configuration = InnReachFeignClientConfig.class)
```

Comment:
this will affect contribution error logging as `InnReachFeignClientConfig` uses custom `FeignDecoder` which is not informative enough (all it provides is response status)

Reply:
I'll create another configuration, w/o `FeignDecoder`, as a temp fix